validphys2/src/validphys/gridvalues.py

```diff
@@ -123,7 +128,7 @@ def evaluate_luminosity(pdf_set: LHAPDFSet, n: int, s:float, mx: float,
     channel: The channel tag name from LUMI_CHANNELS.
     """
 
-
+    # fmt: off
```

Comment:
why did you put this here? I agree that I don't especially like what black does with these long lines but also don't really think it's that much of a problem.

Reply:
I didn't like the changes. Like it is right now they are ordered as

```
   = a1*b1
   + a2*b2
```

which makes sense to keep imho